content/docs/sampling.md

```diff
@@ -50,4 +62,6 @@ Example `strategies.json`
 }
 ```
 
-`service_strategies` defines service specific sampling strategies. There are 2 types of strategies possible: `probabilistic` and `ratelimiting` which are described [above](#client-sampling-configuration). `default_strategy` defines the catch-all sampling strategy that is propagated if the service is not included as part of `service_strategies`.
+`service_strategies` defines service specific sampling strategies and `operation_strategies` defines operation specific sampling strategies. There are 2 types of strategies possible: `probabilistic` and `ratelimiting` which are described [above](#client-sampling-configuration) (NOTE: `ratelimiting` is not supported for `operation_strategies`). `default_strategy` defines the catch-all sampling strategy that is propagated if the service is not included as part of `service_strategies`.
```

Comment:
> (NOTE: `ratelimiting` is not supported for `operation_strategies`)

kind of confusing, maybe say that it's supported for service but not for operations